services/wallet/api.go

```diff
@@ -332,26 +348,31 @@ func (api *API) GetOpenseaAssetsByOwnerAndCollection(ctx context.Context, chainI
 	return container.Assets, nil
 }
 
+// @deprecated
 func (api *API) GetCollectiblesByOwnerAndCollectionWithCursor(ctx context.Context, chainID wcommon.ChainID, owner common.Address, collectionSlug string, cursor string, limit int) (*thirdparty.FullCollectibleDataContainer, error) {
 	log.Debug("call to GetCollectiblesByOwnerAndCollectionWithCursor")
 	return api.s.collectiblesManager.FetchAllAssetsByOwnerAndCollection(chainID, owner, collectionSlug, cursor, limit)
 }
 
+// @deprecated
 func (api *API) GetCollectiblesByOwnerWithCursor(ctx context.Context, chainID wcommon.ChainID, owner common.Address, cursor string, limit int) (*thirdparty.FullCollectibleDataContainer, error) {
```

Comment:
@dlipicar I see this and the following ar still referenced on the Nim side; not used, though. Is there any plan to still use them in the future?
- GetCollectiblesByOwnerAndContractAddressWithCursor
- GetCollectiblesByUniqueID

Reply:
Yes referenced but are still deprecated i think